### DISCUSSION TITLE
https://vuepress-theme-hope.github.io/v2/zh/guide/feature/comment.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#824] [Announcements]
# https://vuepress-theme-hope.github.io/v2/zh/guide/feature/comment.html

通过内置 [@vuepress/plugin-comment][comment]，vuepress-theme-hope 实现了评论功能。 相关信息 vuepress-theme-hope 将主题选项中的 plugins.comment 作为插件选项提供给 @vuepress/plugin-comment。

https://vuepress-theme-hope.github.io/v2/zh/guide/feature/comment.html

**Comment by Tiang-88** [comment]
和教练叫
